app/api/resources/metadata.py

```diff
@@ -238,3 +239,19 @@ async def get_genome_dataset_attributes(
     except Exception as ex:
         logging.error(ex)
         return response_error_handler({"status": 500})
+
+@router.get("/genomeid/{assembly_accession_id}", name="genome_id")
```

Comment:
This is probably not a great pathname. Something like `/assembly/{assembly_accession_id}/genome_id/` would be closer to what is happening here. We may want to discuss this collectively.